Configuration/Generator/python/OmegaMinus_13p6TeV_SoftQCDInel_TuneCUEP8M1_cfi.py

```diff
@@ -0,0 +1,38 @@
+# Found 37 output events for 40000 input events.
+# Filter efficiency = 0.000925
+# Timing = 0.015818 sec/event
+# Event size = 456.1 kB/event
+
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
Following the guideline (*), the generator filter does not include ExternalDecay, should Pythia8ConcurrentGeneratorFilter be used instead?

(*) https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookGenMultithread#Pythia8ConcurrentGeneratorFilter

Reply:
By the way, why TuneCUEP8M1? Not CP5?

Reply:
>  By the way, why TuneCUEP8M1? Not CP5?

was this addressed? @mtosi

Reply:
Yes. Please move to Run3 tuning, Thanks.